pkgs/tools/text/mawk/default.nix

```diff
@@ -1,11 +1,14 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "mawk-1.3.4-20161120";
+  name = "mawk-1.3.4-20171017";
 
   src = fetchurl {
-    url = "ftp://invisible-island.net/mawk/${name}.tgz";
-    sha256 = "07hc33g2ip7463dravsiz0zwfc8vy4y4jxqvp7rz3hb896xw27in";
+    urls = [
+      "ftp://invisible-island.net/mawk/${name}.tgz"
```

Comment:
The ftp site seems super slow to me. Perhaps change the order of these so that the https site below comes first?